pkg/rego/schemas/cloud.json

```diff
@@ -2364,6 +2364,10 @@
           "type": "object",
           "$ref": "#/definitions/github.com.aquasecurity.defsec.pkg.types.IntValue"
         },
+        "deletionprotection": {
```

Comment:
This change to the schema is autogenerated, using `make schema` once the above struct has been changed.